src/koheesio/models/reader.py

```diff
@@ -2,13 +2,11 @@
 Module for the BaseReader class
 """
 
+from typing import Optional
 from abc import ABC, abstractmethod
-from typing import Optional, TypeVar
 
 from koheesio import Step
-
-# Define a type variable that can be any type of DataFrame
-DataFrameType = TypeVar("DataFrameType")
+from koheesio.spark import DataFrame
```

Comment:
I'd like to not have Spark in the BaseReader - this is supposed to work without any trace of Spark

Reply:
Fixed